### PR TITLE
[TACHYON-1099] create/loadMeta return TachyonFile

### PR DESCRIPTION
**Pull request by andreareale**
https://tachyon.atlassian.net/browse/TACHYON-1099

As suggested in PR #1717, the return type of create and loadMeta for TachyonFileSystemCore has been changed to TachyonFile for uniformity.

### CODE REVIEW

**Comment by jsimsa** [comment]
LGTM +1

**Comment by yupeng9** [comment]
LGTM

**Comment by yupeng9** [comment]
@calvinjia , can you take a look?

File: clients/unshaded/src/main/java/tachyon/client/file/AbstractTachyonFileSystem.java [hunk 1]
```diff
@@ -65,12 +65,13 @@ protected AbstractTachyonFileSystem() {
...
 
   @Override
-  public long create(TachyonURI path, CreateOptions options)
+  public TachyonFile create(TachyonURI path, CreateOptions options)
       throws FileAlreadyExistsException, IOException, InvalidPathException, TachyonException {
     FileSystemMasterClient masterClient = mContext.acquireMasterClient();
     try {
-      return masterClient.create(path.getPath(), options.getBlockSize(), options.isRecursive(),
-          options.getTTL());
+      final long id = masterClient.create(path.getPath(), options.getBlockSize(),
```
Comment (calvinjia) [inline clients/unshaded/src/main/java/tachyon/client/file/AbstractTachyonFileSystem.java]: Please name this `fileId` to be consistent

**Comment by calvinjia** [comment]
@andreareale thanks for the patch, left a minor comment. Could you also resolve the merge conflicts?

**Comment by andreareale** [comment]
Merged, thanks

**Comment by calvinjia** [comment]
LGTM

**Pull request merged**